scheduler/scheduler.py

```diff
@@ -2341,9 +2341,11 @@ def _update_priorities(self):
                 else:
                     job_time_so_far = \
                         self._job_time_so_far[job_id][worker_type]
-                    if not self._simulate and job_id in elapsed_job_time:
-                        job_time_so_far += \
-                            elapsed_job_time[job_id][worker_type]
+                    if not self._simulate:
+                        if (job_id in elapsed_job_time and
+                            worker_type in elapsed_job_time[job_id]):
+                            job_time_so_far += \
+                                elapsed_job_time[job_id][worker_type]
```

Comment:
The job may have already completed?

Reply:
Maybe but that's handled earlier, this is just because a job might not have run on every worker type